website/docs.md

```diff
@@ -1890,6 +1890,18 @@ By default, all listeners are **private**, i.e., accessible only from the cloud
 project's internal network. In our example, we declare that the `hello` listener
 is public.
 
```

Comment:
Thanks for bringing this up, we forgot to add a section on health-checks (see TODO above). Could you delete that TODO?

Reply:
deleted